src/ds3.h

```diff
@@ -18,7 +18,6 @@
 
 #include <stdint.h>
 #include <string.h>
-#include <curl/curl.h>
```

Comment:
ahh, OK.  I thought we were trying to separate the dependency, but that makes sense.  I've never tried to build the browser so I'll work through that this morning but I would think it would need to include curl itself if it uses it directly.